src/Sarif.Viewer.VisualStudio.UnitTests/ConverterResourceTests.cs

```diff
@@ -0,0 +1,46 @@
+﻿// Copyright (c) Microsoft. All rights reserved. 
+// Licensed under the MIT license. See LICENSE file in the project root for full license information. 
+
+namespace Microsoft.Sarif.Viewer.VisualStudio.UnitTests
+{
+    using System;
+    using System.Globalization;
+    using System.Linq;
+    using System.Reflection;
+    using FluentAssertions;
+    using Microsoft.CodeAnalysis.Sarif.Converters;
+    using Xunit;
+
+    public class ConverterResourceTests : SarifViewerPackageUnitTests
+    {
+        [Fact]
+        public void VerifyToolFormatOpenLogFileResourcesExist()
+        {
+            FieldInfo[] toolFormatFieldInfos = typeof(ToolFormat).GetFields();
+            foreach (FieldInfo fieldInfo in toolFormatFieldInfos)
+            {
+                string filterString = Resources.ResourceManager.GetString(string.Format(CultureInfo.InvariantCulture, "{0}{1}{2}", OpenLogFileCommands.FilterResourceNamePrefix, fieldInfo.Name, OpenLogFileCommands.FilterResourceNameSuffix), CultureInfo.CurrentCulture);
+                filterString.Should().NotBeNullOrEmpty();
+            }
+        }
+
+        [Fact]
+        public void VerifyOpenLogFileResourceExistInToolFormat()
+        {
+            foreach (PropertyInfo resourcePropertyInfo in typeof(Resources).GetProperties())
+            {
+                // The SARIF filter in the open log dialog is added explicitly and is not in the tool format
+                // class.
```

Comment:
Perhaps more relevant to the actual implementation would be "The resource "ImportNoneFilter" corresponds to the "SARIF files" filter string."